bin/remixdCLI.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+const path = require('path');
+const app = require('express')();
+const ws = require('express-ws')(app);
+const cors = require('cors');
+//const server = require('http').createServer(app)
+//const io = require('socket.io')(server)
+const bodyParser = require('body-parser');
+const dotenv = require('dotenv');
+
+
+// Middleware
+app.use(bodyParser.urlencoded({extended: true}))
+app.use(bodyParser.json());
+app.use(cors({origin: true}));
+app.options('*', cors());
```

Comment:
It seems to me that your are accepting all cors. Here is the list of origins we support out of the box for Remix plugins : https://github.com/ethereum/remix-plugin/blob/master/projects/client/assets/origins.json

Reply:
Cors should be extended depending on the input of the user

Reply:
Basically another CORS option would be --remix-ide flag so on top of the listed one that one should be added. Other ways that you would support this?

Reply:
If I understand well you would fetch the list of links, then you would append the `--remix-ide` value to it, and set the cors with these links ?
Do you think it's an issue if you have to make an http request to the list of authorized origin ?

Reply:
One problem I found out is that CORS in not directly supported by express so I've added check if current origin is in the list of whitelisted origins. 

I can make a http request everytime on start of the remixd (that should be enough).

I can querry directly https://github.com/ethereum/remix-plugin/blob/master/projects/client/assets/origins.json (or some other github file)

Reply:
I've added https://github.com/ethereum/remixd/pull/92/files#diff-48ed5ebce7c27013a09b99e149ab3542 to query the url and save it locally. 

When starting the server it always reads last version from origins.json file and query is seperate task.

Reply:
This has also been resolved.